Core/BookmarksCoreDataStorage.swift

```diff
@@ -795,10 +803,17 @@ extension BookmarksCoreDataStorage {
             
             let results = try? viewContext.fetch(fetchRequest)
             guard (results?.count ?? 0) <= 1 else {
+                let count = results?.count ?? 0
+                let pixelParam = [PixelParameters.bookmarkFolderCount: "\(count)"]
+
+                Pixel.fire(pixel: .debugFavoriteOrphanFolder, withAdditionalParameters: pixelParam)
+                Thread.sleep(forTimeInterval: 1)
```

Comment:
For the record, I'm horrified 😅

Reply:
Stolen from here: 
https://github.com/duckduckgo/iOS/blob/83e3ffed1f813f0f4ad3512ccd3355ab3107eecc/Core/Database.swift#L71

But hopefully this will be removed soon

Reply:
Maybe we need a custom method to hide our sins, like "fatelErrorWithPixel(_: )